tests/behat/uploadFileToMoodle.feature

```diff
@@ -43,3 +43,18 @@ Feature: upload the resource in oCIS to moodle
     And I click on "//*[@class='fp-filename-field']/p[text()='testfile.txt']" "xpath_element"
     And I click on "Select this file" "button"
     Then I should see "testfile.txt"
+  @skipOnStable
```

Comment:
For stable branch, disabling sync would result in 200 status code. This was bug in master and was fixed but it look like it wasn't backported in stable. Thus, skipping this test in stable

Reply:
maybe better just to make the test more relaxed, here we don't really care or test what ocis exactly responds, just if it works for the user.